notebooks/243-named-entity-recognition/243-named-entity-recognition.ipynb

```diff
@@ -0,0 +1,212 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #31.    optimized_model = OVModelForTokenClassification.from_pretrained(save_dir)
</pre><p>can we compare accuracy and performance optimized model and original one?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1163/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>